python/openassetio/hostAPI/Manager.py

```diff
@@ -557,105 +557,105 @@ def entityDisplayName(self, entityRefs, context):
 
     @debugApiCall
     @auditApiCall("Manager methods")
-    def getEntityMetadata(self, entityRefs, context):
+    def getEntityAttributes(self, entityRefs, context):
         """
-        Retrieve @ref metadata for each given entity. This may contain
-        well-known keys that you can then use to further customize your
-        handling of the entity. Some types of entity may only have
-        metadata, and no meaningful @ref primary_string.
+        Retrieve @ref attributes for each given entity. This may contain
+        well-known attributes that you can then use to further customize
+        your handling of the entity. Some types of entity may only have
+        attributes, and no meaningful @ref primary_string.
 
-        There are some well-known keys defined in the core API - @see
-        openassetio.constants. These have universally agreed meaning.
+        There are some well-known attribute names defined in the core
+        API - @see openassetio.constants. These have universally agreed
+        meaning.
 
-        As a host, you can also advertize well-known keys of your own as
-        part of any first-class asset based workflows you may have. For
-        example, a compositor may choose to consume the `colorspace`
-        key (if present) and adjust the input space of an image reader
-        node accordingly.
+        As a host, you can also advertize well-known attribute names of
+        your own as part of any first-class asset based workflows you
+        may have. For example, a compositor may choose to consume the
+        `colorspace` attribute (if present) and adjust the input space
+        of an image reader node accordingly.
 
-        These new keys should be clearly explained in your
+        These new attribute names should be clearly explained in your
         documentation. A manager may then be able to provide this
         information, based on introspection of your identifier (@see
```

Comment:
[consider]/[plan]  Another case unrelated to this PR where the rendered output doesn't match what the text clearly expects
![image](https://user-images.githubusercontent.com/7827094/144602776-a5d87892-6064-4681-9348-708748c336ee.png)

Reply:
#128